README.md

```diff
@@ -6,7 +6,7 @@ This gem provides a convenient interface for creating releases and deploying usi
 
 ### Branch Notes
 
-* `master` should only contain deployable code.
+* `master` / `main` should only contain deployable code.
```

Comment:
I think I'd prefer to prioritise `main` over `master` when we refer to the primary branch